### PR TITLE
Package refactor

### PR DESCRIPTION
**Pull request by cgtobi**
This PR restructures the package to only use the pyatmo namespace and allow for testing against the actual package rather than the local code. It also introduces tox to automatically test against different python version and pre-commit hooks to check the commit with black, isort and flake8.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Note: don't use this config for your own repositories. Instead, see
+# "Version control integration" in README.md.
+exclude: ^(fixtures/)
+repos:
+-   repo: local
+    hooks:
+    - id: black
```
Comment (jabesq) [inline .pre-commit-config.yaml]: I would prefer that pre-commit uses its own version of tools an not the local ones like this for black:
```
- repo: https://github.com/psf/black
  rev: stable
  hooks:
    - id: black
      language_version: python3.6
```

File: README.md [hunk 2]
```diff
@@ -22,26 +23,47 @@ Once installed you can simple add `pyatmo` to your Python 3 scripts by including
...
+Testing
+-------
 
 To run the pytest testsuite you need to install the following dependencies:
 
     pip install pytest pytest-mock pytest-cov requests-mock freezegun
 
+`pyatmo` has to be installed in editable mode to be picked up by pytest. Simply run:
+
+    pip install -e .
```
Comment (jabesq) [inline README.md]: pyatmo is already configured to work with pipenv so install can be reduce to `pipenv install`and it's create a virtual environment with pyatmo in it.

Comment (cgtobi) [reply README.md]: Ok, great. I have no experience with `pipenv` so I take a look asap.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (cgtobi) [reply .pre-commit-config.yaml]: How about using the version from within the virtual environment? I'd like that better than installing yet another version of a tool alongside.

**Pull request merged**